src/main/java/com/wojtek/parkingmeter/bootstrap/Bootstrap.java

```diff
@@ -1,49 +1,94 @@
 package com.wojtek.parkingmeter.bootstrap;
 
-import com.wojtek.parkingmeter.helpers.TicketType;
-import com.wojtek.parkingmeter.modelDAO.ParkingOperatorDAO;
-import com.wojtek.parkingmeter.modelDAO.ParkingOwnerDAO;
-import com.wojtek.parkingmeter.modelDAO.TicketDAO;
-import com.wojtek.parkingmeter.repositories.ParkingOperatorRepository;
-import com.wojtek.parkingmeter.repositories.ParkingOwnerRepository;
+import com.wojtek.parkingmeter.helpers.ChargeCalculator;
+import com.wojtek.parkingmeter.helpers.enums.TicketType;
+import com.wojtek.parkingmeter.model.CarEntity;
+import com.wojtek.parkingmeter.model.TicketEntity;
+import com.wojtek.parkingmeter.repositories.CarRepository;
 import com.wojtek.parkingmeter.repositories.TicketRepository;
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.stereotype.Component;
 
-import java.sql.Timestamp;
+import java.time.LocalDateTime;
 
 @Component
 public class Bootstrap implements CommandLineRunner {
 
-    private final ParkingOwnerRepository parkingOwnerRepository;
-    private final ParkingOperatorRepository parkingOperatorRepository;
     private final TicketRepository ticketRepository;
+    private final CarRepository carRepository;
+
+    public Bootstrap(TicketRepository ticketRepository, CarRepository carRepository) {
 
-    public Bootstrap(ParkingOwnerRepository parkingOwnerRepository, ParkingOperatorRepository parkingOperatorRepository, TicketRepository ticketRepository) {
-        this.parkingOwnerRepository = parkingOwnerRepository;
-        this.parkingOperatorRepository = parkingOperatorRepository;
         this.ticketRepository = ticketRepository;
+        this.carRepository = carRepository;
     }
 
 
     @Override
     public void run(String... args) throws Exception {
 
+        load_tickets();
+
+        load_cars();
+    }
+
+    private void load_cars() {
```

Comment:
dlaczego to jest w kodzie produkcyjnym a nie testowym?

Reply:
Przemyślę to i rozdzielę. ;)